resources/public/api/conf/1.0/schemas/retrieveBusinessDetails/retrieve_business_details_response.json

```diff
@@ -0,0 +1,95 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Retrieve additional information for a user's business details",
+  "description": "Retrieve business details response",
+  "type": "object",
+  "properties": {
+    "businessId": {
+      "type": "string",
+      "description": "A unique identifier for the business income source.",
+      "example": "XAIS12345678910"
+    },
+    "typeOfBusiness": {
+      "type": "string",
+      "description": "The type of business income source, one of: 'self-employment', 'uk-property' or 'foreign-property'.",
```

Comment:
Probably will look better if you had the 3 income sources with the structure of the RAML Enum

Reply:
Example

`"enum": [
      "se",
      "up"
]`